WebDriverFactory/src/test/java/DriverManager.java

```diff
@@ -0,0 +1,34 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+
+public abstract class DriverManager {
+	
+	private WebDriver driver;
+
+	public abstract void createDriver();
+
+	public WebDriver getDriver() {
+		createDriver();
+		return this.driver;
+	}
+
+	protected void setDriver() {
+		var chromeDriverManager = new ChromeDriverManager();
```

Comment:
Abstract classes should be general. There should be no mention of concrete DriverManager classes here (ChromeDriverManager).